apps/financial-aid/web-veita/src/components/NewUserModal/NewUserModal.tsx

```diff
@@ -148,44 +156,48 @@ const NewUserModal = ({ isVisible, setIsVisible, onStaffCreated }: Props) => {
         Notandi fær sendan tölvupóst með hlekk til að skrá sig inn með rafrænum
         skilríkjum.
       </Text>
-      <Text marginBottom={3} variant="h4">
-        Réttindi notanda
-      </Text>
-      <Box marginBottom={3}>
-        <Checkbox
-          name={'employee'}
-          label="Vinnsluaðili"
-          checked={state.roles.includes(StaffRole.EMPLOYEE)}
-          strong={false}
-          hasError={state.hasError && state.roles.length === 0}
-          onChange={(event) => {
-            changeStaffAccess(StaffRole.EMPLOYEE, event.target.checked)
-          }}
-        />
-      </Box>
-      <Box marginBottom={2}>
-        <Checkbox
-          name={'admin'}
-          label="Stjórnandi (admin)"
-          checked={state.roles.includes(StaffRole.ADMIN)}
-          hasError={state.hasError && state.roles.length === 0}
-          onChange={(event) => {
-            changeStaffAccess(StaffRole.ADMIN, event.target.checked)
-          }}
-          strong={false}
-        />
-      </Box>
+      {showUserRights && (
```

Comment:
I think you would never get through, because the validation areRequiredFieldsFilled and roles is required 🏗️ ??